src/components/mnemonic/mnemonic-input/mnemonic-input.tsx

```diff
@@ -46,6 +47,7 @@ export const MnemonicInput: FC<MnemonicProps> = ({
         onBlur={handleBlur}
         onFocus={() => setIsFocused(true)}
         onChangeText={onChangeText}
+        testID={testID}
```

Comment:
do we need here `setTestId()`?

Reply:
looks like `react native` `0.64` supports `testId` as well

https://stackoverflow.com/a/68746717/6559770

Reply:
`setTestId` is applied to `react-native` components (ex `TextInput`, `TouchableOpacity`).  
here I'm just drilling `testID` prop to the native componnets

Reply:
I have tried settings up `testID` props without `accessibilityLabel` - it does not work